cmd/cmd_utils.go

```diff
@@ -423,17 +425,80 @@ func printMessageForMissingAtmosConfig(cliConfig schema.CliConfiguration) {
 
 // printMessageToUpgradeToAtmosLatestRelease prints info on how to upgrade Atmos to the latest version
 func printMessageToUpgradeToAtmosLatestRelease(latestVersion string) {
```

Comment:
I think we implemented this the _hard way_, and instead, we should use Lipgloss everywhere. Please reimplement using bubbletea or lipgloss.

https://github.com/charmbracelet/bubbletea/tree/main/examples

https://github.com/charmbracelet/lipgloss

Reply:
e.g.

```
package main

import (
	"fmt"
	"strings"

	"github.com/charmbracelet/lipgloss"
)

func printMessageToUpgradeToAtmosLatestRelease(currentVersion, latestVersion string) {
	// Define styles
	headerStyle := lipgloss.NewStyle().
		Foreground(lipgloss.Color("12")).
		Bold(true).
		Align(lipgloss.Center)
	messageStyle := lipgloss.NewStyle().
		Foreground(lipgloss.Color("10")).
		Margin(1, 0)
	linkStyle := lipgloss.NewStyle().
		Foreground(lipgloss.Color("14")).
		Underline(true)
	boxStyle := lipgloss.NewStyle().
		Border(lipgloss.RoundedBorder()).
		BorderForeground(lipgloss.Color("6")).
		Padding(1, 2).
		Width(80)

	// Message content
	header := headerStyle.Render("Update Available!")
	message := messageStyle.Render(fmt.Sprintf("You are using version %s. The latest version is %s.", currentVersion, latestVersion))
	links := []string{
		linkStyle.Render("Atmos Releases: https://github.com/cloudposse/atmos/releases"),
		linkStyle.Render("Install Atmos: https://atmos.tools/install"),
	}
	linksRendered := strings.Join(links, "\n")

	// Combine everything into a single box
	content := fmt.Sprintf("%s\n\n%s\n\n%s", header, message, linksRendered)
	box := boxStyle.Render(content)

	// Print the box
	fmt.Println(box)
}

func main() {
	printMessageToUpgradeToAtmosLatestRelease("1.2.3", "1.3.0")
}
```

Reply:
@osterman , sure, no worries, just pushed the changes with lipgloss